README.md

```diff
@@ -1,2 +1,191 @@
+
+The build is minified and the filenames include the hashes.
+Your app is ready to be deployed!
+
+### `npm run start`
+
+Runs the app in production mode. Assumes a build was already produced before by the `build` script.
+
+### `npm run export`
+
+Exports the app to be served statically into the `out` folder. Assumes a build was already produced before by the `build` script.
+
+### `npm run lint`
+
+Lints all the project JS and CSS files, ensuring they match the coding-standards.
+
+### `npm run test`
+
+Run the test files found in the `src/` folder.
+
+#### Options
+
+Every option is passed to the `jest` command, however here are the most useful ones:
+
+##### `<expression>`
+
+If passed an expression, **Jest** will compare every file with that expression and only run tests of those whose match is positive.
+
+##### `--coverage`
+
+After tests, it outputs a table with the code coverage.
+If the coverage is below the target, it throws an error.
+
+This option is passed when tests are ran in the pipeline, which means that if the coverage is below the projects target, the pipeline will fail.
+
+##### `--watch`
+
+This options will not terminate the process once tests are complete and will observe any file changes and if detected, re-run the tests.
+
+##### `-u`
+
+This options will update every existing snapshot test with the result of the new snapshot tests.
+
+#### Example
+
+`npm run test -- App --coverage -u`
+
+This example will run tests for files like `App.test.js` or `SpecialApp.test.js`, display the coverage (for the matched files) and update any existing snapshot.
+
+## Environment variables
+
+All environment variables will be be available to the server-side but only variables that start with `REACT_APP_` will be accessible to the client-side. This is made intentionality as a security measure, so that environment variables are exposed selectively. These will be embeded at **build time**, thus are **read-only**. This means you must rebuild your application every time you change them.
+
+Here's the list of used **build-time** environment variables:
+
+- `NODE_ENV`: One of `development` or `production` (automatically set).
+- `REACT_APP_GA_TRACKING_ID`: The Google Analytics tracking ID to be used (if empty, tracking will be disabled).
+
+Besides these variables, you may also define the `HOST` and `PORT` environment variables, which will be used to bind the server.
+
+## Deployment
+
+### Staging
+
+There's a CI/CD pipeline configured for staging, allowing you to quickly preview outcome of changes.
+
+If the build succeeds, any commit will be deployed on [now](https://zeit.co/now) and will have a unique URL for you to preview the deployment. Moreover, branches have fixed URLs, meaning they will stay the same even if they change over time in terms of commits.
+
+These URLs will are available directly on GitLab, thanks to `now`'s tight integration.
```

Comment:
I think we shouldn't advice this integration since it requires the connection between the project and a team member. The integration doesn't work with `now` teams so we've been using a CI configuration to deploy with a generic token and not in the name of a team member.

Reply:
If we remove now.json, we should remove this section altogether. Perhaps setting up now should be a recipe.

Reply:
I agree, this should be a recipe. Where do you think we should put recipes?
Should this repo have a folder like `boilerplate` or `next-with-moxy` where it has all necessary files and a folder name `recipes` where it has a README pointing to specific recipe READMEs?

Reply:
After speaking with @satazor and @ivolimasilva, we will start by creating a `recipes` folder in this repo with all recipes. Together will also be a `boilerplate` folder.

Reply:
Done